cetmix_tower_aws/readme/USAGE.md

```
@@ -0,0 +1,112 @@
+Instance ID: i-0abc123def456789, Type: t2.micro, State: running
+Instance ID: i-0def456abc789123, Type: t3.medium, State: stopped
+Instance ID: i-0789abc123def456, Type: m5.large, State: running
+```
+
+For a successful execution with no EC2 instances:
+
+```bash
+No EC2 instances found
+```
+
+**Creating Additional AWS Commands**
```

markdownlint-cli2 (0.17.2)

Emphasis used instead of a heading
null

(MD036, no-emphasis-as-heading)